release/README.md

```diff
@@ -2,8 +2,37 @@
 
 This directory contains tools for Kata Containers releases.
 
+## update-repository-version.sh ##
+
+This script is used to create a Github pull request (a.k.a PR) to change version in all
+the Kata repositories.
+
+To get more information about how to use it , run:
+
+```bash
+$ ./update-repository-version.sh -h
+```
+
+### Update Kata projects to a new version ###
+Kata Containers is devided in multiples projects. On each release all the
+projects version is updated to keep the same version in all of them.
+
+To update all the projects versions use:
+
+```bash
+$ make bump-kata-version NEW_VERSION=<new-version>
+```
+
+The makefile target `bump-kata-version` will create a Gihub pull request
```

Comment:
Lines 26-29 suggested rewrite: The makefile target `bump-kata-version` creates a GitHub pull request in the kata repositories. These pull requests are tested by the Kata CI to ensure the entire project is working prior to the release. Next, the PR is approved and merged by Kata Containers members.